lib/fs/__tests__/multi-param.test.js

```diff
@@ -152,3 +154,59 @@ describe('fs.write()', () => {
     })
   })
 })
+
+describeNode12('fs.writev()', () => {
+  let TEST_FILE
+  let TEST_DATA
+  let TEST_FD
+
+  beforeEach(() => {
+    TEST_FILE = path.join(os.tmpdir(), 'fs-extra', 'writev-test-file')
+    TEST_DATA = [crypto.randomBytes(SIZE / 2), crypto.randomBytes(SIZE / 2)]
+    fs.ensureDirSync(path.dirname(TEST_FILE))
+    TEST_FD = fs.openSync(TEST_FILE, 'w')
+  })
+
+  afterEach(() => {
+    return fs.close(TEST_FD)
+      .then(() => fs.remove(TEST_FILE))
+  })
+
+  describe('with promises', () => {
+    it('returns an object', () => {
+      return fs.writev(TEST_FD, TEST_DATA, 0)
+        .then(({ bytesWritten, buffers }) => {
+          assert.strictEqual(bytesWritten, SIZE, 'bytesWritten is correct')
```

Comment:
I believe the message should be the opposite, something like `bytesWritten is incorrect` since that is used as the message for `AssertionError` if the values are not strictly equal. It is the same for other assertion messages down below.

https://nodejs.org/api/assert.html#assert_assert_strictequal_actual_expected_message

Reply:
Yeah, perhaps; I don't think it really matters, the main point is to clearly identify which assertion is throwing if you have an error. In any case, these are simply copied from tests earlier in the file, so if we want to fix this, it should be in a separate PR that fixes the entire file.